test/client.test.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright 2020-2022 SubQuery Pte Ltd authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import { NetworkClient } from '../packages/network-clients/src';
+import { NetworkClient } from "../packages/network-clients/src";
```

Comment:
Can change these 2 double quote to single quote, will need add better prettier rules